pom.xml

```diff
@@ -121,7 +121,7 @@
     <nuxeo.webengine.version>8.3-SNAPSHOT</nuxeo.webengine.version>
     <nuxeo.features.version>8.3-SNAPSHOT</nuxeo.features.version>
 
-    <nuxeo.connect.client.version>1.4.24</nuxeo.connect.client.version>
+    <nuxeo.connect.client.version>1.4.25-SNAPSHOT</nuxeo.connect.client.version>
```

Comment:
Not a pain: one line two uncomment in the same file.

It is allowed on development branches, not on master or maintenance branches. And certainly not on release.

SNAPSHOT dependencies are usually a mistake, coming from a transitive path or a forgotten work in progress and lead to random errors. It also increases the workload and the risk at release time when it comes mandatory to replace them with a release version in order to satisfy the most important rule: **a release build must be reproducible**.
That's why the default rule is not to use SNAPSHOT dependencies on master or maintenance branches, except during one development cycle, on purpose, during their own parallel development and knowing that they will have to be replaced with a release version before the release of the current code.
So, for an average of three or four library cases over all the code, it's worth to explicitly mention those very few exceptions for which the tooling will not raise an alert.

Please read https://doc.nuxeo.com/display/CORG/Maven+Coding+Rules#MavenCodingRules-SNAPSHOTdependencies